pkg/trace/README.md

```diff
@@ -62,8 +62,10 @@ This stores the data locally in the specified directory.
 ### Push Based Event Collection
 
 Push based event collection is where the consensus node pushes trace data to an
-external server. At the moment, this is just an S3 bucket. To use this, add the
-following to the config.toml file:
+external server. At the moment, this is just an S3 bucket. To use this, two options are available:
+#### Using push config file
```

Comment:
[nit]

Maybe also add the new lines in this PR too :D

Reply:
I actually mirrored the changes in the other PR in here as well https://github.com/celestiaorg/celestia-core/pull/1319/commits/01b1852bf939aa88e021bc9bb9d6f5014b0984c0 
Is any other new line missing?

Reply:
Should also have a line before the heading, didn't notice the previous PR doesn't have it, but it's fine, just a nit :D